Code.gs

```diff
@@ -225,7 +223,11 @@ function main(){
 
           if(e.getStartTime().getTime() != fe.startTime.getTime() ||
              e.getEndTime().getTime() != fe.endTime.getTime())
-            e.setTime(fe.startTime, fe.endTime)
+            if (fe.isAllDay){
```

Comment:
I assume all the 

```
  if (fe.isAllDay){
    e.setAllDayDates(fe.startTime, fe.endTime);
  }else{
    e.setTime(fe.startTime, fe.endTime);
  }
```

is supposed to be within the parent if statement? I'm not super fluent in JS, but do we need brackets around it then? If not, maybe just add them for clarity (also did some minor corrections to conform with conventions in the rest of the program):

```
if(e.getStartTime().getTime() != fe.startTime.getTime() ||
   e.getEndTime().getTime() != fe.endTime.getTime()){
  if (fe.isAllDay)
    e.setAllDayDates(fe.startTime, fe.endTime);
  else
    e.setTime(fe.startTime, fe.endTime);
}
```

Reply:
> is supposed to be within the parent if statement?
Correct.

Pretty sure brackets are not needed but adding them for clarity is always a good idea

Reply:
Right. I'm writing up a "coding conventions" document for the wiki right now. Go ahead and push another commit with the suggested change and I'll approve it